pkgs/applications/video/handbrake/default.nix

```diff
@@ -96,6 +92,8 @@ stdenv.mkDerivation rec {
     substituteInPlace libhb/module.defs \
       --replace '$(CONTRIB.build/)include/libxml2' ${libxml2.dev}/include/libxml2
 
+    substituteInPlace gtk/configure.ac --replace 'HB_LIBS="-lhandbrake -lavformat -lavfilter -lavcodec -lavutil -ldav1d -lswresample -lpostproc -ldvdnav -ldvdread -lmp3lame -lvorbis -lvorbisenc -logg -lx264 -lswscale -ltheoraenc -ltheoradec -lvpx -lz -lbz2 -lbluray -lass -lfontconfig -lfreetype -lxml2 -ljansson -lopus -lspeex -llzma"' 'HB_LIBS="-lhandbrake -lavformat -lavfilter -lavcodec -lavutil -ldav1d -lswresample -lpostproc -ldvdnav -ldvdread -lmp3lame -lvorbis -lvorbisenc -logg -lx264 -lswscale -ltheoraenc -ltheoradec -lvpx -lz -lbz2 -lbluray -lass -lfontconfig -lfreetype -lxml2 -ljansson -lopus -lspeex -llzma -ldl"'
```

Comment:
I'm happy to do something else. I didn't add to `NIX_LDFLAGS` because it's one specific part of the build that's not working for us: the GTk-using `ghb` executable. The library is mentioned for that build [here](https://github.com/HandBrake/HandBrake/blob/master/gtk/configure.ac#L256), but I'm not sure how our build is not working as expected, so I opted for something to minimize the patching. In that respect, I'd prefer `NIX_LDFLAGS`, too.

Reply:
That line is only a fallback branch. For us, it would have to be in the `*-*-linux*` branch. And apparently they fixed it sometime after the release:

https://github.com/HandBrake/HandBrake/commit/f7bc55eb2cf06759b2cbe339c858d1d6a6b2d831#diff-3bfc7b7ac6e2f746f2fa2173192d96fdR253